src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/styles/use_workflows_monaco_theme.ts

```diff
@@ -32,6 +32,10 @@ export function useWorkflowsMonacoTheme() {
         'editorHoverWidget.foreground': euiTheme.colors.textParagraph,
         'editorHoverWidget.background': euiTheme.colors.backgroundBasePlain,
         'editorHoverWidget.border': euiTheme.colors.borderBaseSubdued,
+        'editorLineNumber.foreground': euiTheme.colors.textPrimary,
+        'editorLineNumber.activeForeground': euiTheme.colors.textSubdued,
```

Comment:
```suggestion
        'editorLineNumber.foreground': euiTheme.colors.textSubdued,
        'editorLineNumber.activeForeground': euiTheme.colors.textPrimary,
```

@r4zr32d3k1l ask was to change the default color of the line number to subdued, not only the active line, rather the other way around :-)